### PR TITLE
March newsletter

### PR DESCRIPTION
**Pull request by bethaitman**
Newsletter for March! Note to self to add stuff to the index as well.

### CODE REVIEW

**Comment by plaindocs** [comment]
@bethaitman see #808

**Comment by bethaitman** [comment]
Thanks Sam, added the link!

**Comment by plaindocs** [comment]
If you want to use the names and talk titles, take the snippet from lines 20-23 of the announcement, and hard code the variables, `2019`, `portland`, etc.

**Review by hillaryfraley — APPROVED** [review]
All looks good to me!

File: docs/blog/newsletter-march-2019.rst [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+######################################
+Write the Docs Newsletter – March 2019
+######################################
+
+Hi everyone! Beth here, with your March newsletter. How on earth is February over already?
+
+We've got lots of conference news to share with you. Potential speakers, you'll be excited to hear that the `Prague Call for Proposals </conf/prague/2019/news/cfp-open/>`_ is open. Get working on those submissions - they need to be in by midnight (CEST) on Tuesday 30 April.
+
+And there's even more going on in Europe: `a new conference in Vilnius, Lithuania </conf/vilnius/2019/news/init/>`_! It'll be two days, 2-3 June; the CFP and ticket sales are opening soon.
```
Comment (plaindocs) [inline docs/blog/newsletter-march-2019.rst]:
```suggestion
And there's even more going on in Europe: `a new conference in Vilnius, Lithuania </conf/vilnius/2019/>`_! It'll be two days, 2-3 June; the CFP and ticket sales are opening soon.
```

File: docs/blog/newsletter-march-2019.rst [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+On the other side of the pond, `Portland speakers have been announced </conf/portland/2019/news/announcing-speakers/>`_. If you're considering buying tickets, do it soon so you don't miss out: they sell out earlier every year. Want to go but need to persuade someone senior that it's worthwhile? The "`convince your manager </conf/portland/2019/convince-your-manager/>`_" page might help.
```
Comment (plaindocs) [inline docs/blog/newsletter-march-2019.rst]:
```suggestion
On the other side of the pond, `Portland speakers have been announced </conf/portland/2019/news/announcing-speakers/>`_. If you're considering buying tickets, do it soon so you don't miss out: they sell out earlier every year. Want to go but need to persuade someone senior that it's worthwhile? The `convince your manager </conf/portland/2019/convince-your-manager/>`_ page might help.
```

File: docs/blog/newsletter-march-2019.rst [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+You may also be interested to know that we've updated the `code of conduct </code-of-conduct/>`_. Some sections have been improved and others clarified - for full details, check out the `code of conduct blog post </blog/new-community-coc-2019/>`_.
```
Comment (plaindocs) [inline docs/blog/newsletter-march-2019.rst]:
```suggestion
We've updated our `code of conduct </code-of-conduct/>`_. Some sections have been improved and others clarified - for full details, check out the `code of conduct blog post </blog/new-community-coc-2019/>`_.
```

I don#'t think this is a 'may' kinda scenario.

File: docs/blog/newsletter-march-2019.rst [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+Moving to docs-as-code: static site generators
+----------------------------------------------
+
+It's no secret that many of us are keen on the docs-as-code approach. But as someone asked this month: if you're moving from a traditional authoring tool like Flare, and want to start storing docs in Git, what should you use to create the docs themselves?
+
+Static site generators are a sensible answer. But how to choose one? `There are so many <https://www.staticgen.com/>`_! The community had a few suggestions to help out: one was to check out `O'Reilly's free intro to SSGs <https://learning.oreilly.com/library/view/static-site-generators/9781492048558/>`_. There were also few specific recommendations:
+
+- `Antora <https://antora.org/>`_ (uses AsciiDoc)
+- `Gatsby <https://www.gatsbyjs.org/>`_
+- `Hugo <https://gohugo.io/>`_ (and there are `introductions to Jekyll, Hugo and Sphinx here <https://www.docslikecode.com/learn/>`_)
```
Comment (plaindocs) [inline docs/blog/newsletter-march-2019.rst]: This reads odd to me. Not really connected to Hugo. 

What about adding Hugo, Jekyll and Sphinx as their own items in the list, and then mention the intros at the end?

**Review by plaindocs — COMMENTED** [review]
Minor comments.

**Comment by Bradamant3** [comment]
LGTM. The index is SO AWESOME! ❤️

File: docs/blog/newsletter-march-2019.rst [hunk 5]
```diff
@@ -5,21 +5,23 @@
...
+On to what we've been talking about on Slack this month!
 
 ------------------------------------------
 Roadmaps and documenting 'future features'
 ------------------------------------------
 
-As documentarians, we can come head-to-head with other stakeholders on what to include in our documentation. One contentious topic is forward-looking statements: some stakeholders might want to tell users that a feature is coming soon. But when we mention a future feature in docs, it can be a problem if that feature doesn't end up being delivered.
+As documentarians, we sometimes come head-to-head with other stakeholders on what to include in our documentation. One contentious topic is forward-looking statements: some stakeholders might want to tell users that a feature is coming soon. But when we mention a future feature in docs, it can be a problem if that feature doesn't end up being delivered.
 
 Here are some thoughts about how to approach this:
```
Comment (clairealyssa) [inline docs/blog/newsletter-march-2019.rst]: So it looks like we're missing the list here?

File: docs/blog/newsletter-march-2019.rst [hunk 6]
```diff
@@ -94,7 +96,7 @@ Community events coming up
 
 - 5 March - Austin, TX, USA - `Lunch meetup <https://www.meetup.com/WriteTheDocs-ATX-Meetup/events/258689786/>`_
```
Comment (clairealyssa) [inline docs/blog/newsletter-march-2019.rst]: Do we only include events for the current month (March) here?

Comment (clairealyssa) [reply docs/blog/newsletter-march-2019.rst]: If we include future months, I have an event to add. :)

**Review by clairealyssa — DISMISSED** [review]

File: docs/blog/newsletter-march-2019.rst (hunk 6 above)
Comment (bethaitman) [reply docs/blog/newsletter-march-2019.rst]: I've been going roughly a fortnight into the next month to make sure we don't miss anything. Send me the link and I'll add it to the list!

File: docs/blog/newsletter-march-2019.rst (hunk 4 above)
Comment (bethaitman) [reply docs/blog/newsletter-march-2019.rst]: Yeah, good shout, I'll fix that

File: docs/blog/newsletter-march-2019.rst (hunk 5 above)
Comment (bethaitman) [reply docs/blog/newsletter-march-2019.rst]: I don't quite understand your comment?

File: docs/blog/newsletter-march-2019.rst (hunk 3 above)
Comment (bethaitman) [reply docs/blog/newsletter-march-2019.rst]: yep, agreed!

File: docs/blog/newsletter-march-2019.rst (hunk 5 above)
Comment (clairealyssa) [reply docs/blog/newsletter-march-2019.rst]: Oh, I was looking at the wrong view - it had looked like the bulleted list that you have under "Here are some thoughts about how to approach this" was just absent. But it's there! 👍

**Pull request merged**